internal/mesh/internal/cache/sidecarproxycache/computed_routes_cache.go

```diff
@@ -27,11 +27,6 @@ func (c *ComputedRoutesCache) TrackComputedRoutes(computedRoutes *types.DecodedC
 	for _, pcr := range computedRoutes.Data.PortedConfigs {
 		for _, details := range pcr.Targets {
 			serviceRefs = append(serviceRefs, details.BackendRef.Ref)
-			if details.FailoverConfig != nil {
```

Comment:
This is no longer necessary as I updated the routes-controller to dump all failover destinations into the top level `Targets` map which we already index in this loop.